### PR TITLE
Use batch query for vms and user topological api v1.0

### PR DESCRIPTION
**Pull request by lpichler**
NOTE: it is not possible to use ruby Topological Api Client because batch querying is not working.

@miq-bot assign @agrare 

cc @Ladas (removing TODO with your name :) )

### CODE REVIEW

**Review by Ladas — APPROVED** [review]
Code looks great. Awesome @lpichler :+1: :tada: \o/

One little TODO removed, 999 left. This lsmola guys is just keeping the mall over. :-D

**Comment by lpichler** [comment]
thanks @Ladas (I was removing rubocop warnings.)

**Review by slemrmartin — APPROVED** [review]
:+1: Cool!
Definitely we have to do less-TODO(lsmola) celebration! :laughing:

File: lib/topological_inventory/host_inventory_sync.rb [hunk 1]
```diff
@@ -42,23 +42,23 @@ def run
...
     end
 
+    def build_topological_inventory_url_from(additional_params)
+      additional_params[:path] = File.join(topological_inventory_api_params_hash[:path], additional_params[:path]) if additional_params[:path]
+      URI::HTTP.build(topological_inventory_api_params_hash.merge(additional_params)).to_s
+    end
+
     class << self
-      TOPOLOGICAL_INVENTORY_API_VERSION = "v0.1".freeze
+      TOPOLOGICAL_INVENTORY_API_VERSION = "v1.0".freeze
```
Comment (agrare) [inline lib/topological_inventory/host_inventory_sync.rb:51]: Wow this probably wasn't working for a looooong time

**Review by agrare — APPROVED** [review]

**Pull request merged**